src/main/java/pl/umk/mat/git2befit/converter/LongListConverter.java

```diff
@@ -0,0 +1,25 @@
+package pl.umk.mat.git2befit.converter;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Converter
+public class LongListConverter implements AttributeConverter<List<Long>, String> {
+    private static final String SPLIT_CHAR = ";";
+
+    @Override
+    public String convertToDatabaseColumn(List<Long> longs) {
+        return longs != null ? String.join(SPLIT_CHAR, String.valueOf(longs)) : "";
+    }
+
+    @Override
+    public List<Long> convertToEntityAttribute(String s) {
+        return s != null ? Arrays.stream(s.split(SPLIT_CHAR))
+                .map(Long::parseLong)
+                .collect(Collectors.toList()) : Collections.emptyList();
+    }
```

Comment:
W linii 23 zwracasz pustą Listę natomiast w lini 16 sprawdzasz czy ta List jest nullem, a emptyList() zwraca pustą listę nie null. Więc nie dojdzie do drugiego warunku w lini 16. Czy się mylę?

Reply:
Jeżeli mamy pewność, że wszystkie inserty są robione przez orma to tak. W sytuacji kiedy robimy je ręcznie może być null.